src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -0,0 +1,47 @@
+package codeu.controller;
+
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet class responsible for the activity feed page. */
+public class ActivityFeedServlet extends HttpServlet {
+
+  /** Store class that gives access to Users. */
+  private UserStore userStore;
+
+  /**
+   * Set up state for handling activity feed-related requests. This method is only called when running in a
+   * server, not when running in a test.
+   */
+  @Override
+  public void init() throws ServletException {
+    super.init();
+    setUserStore(UserStore.getInstance());
+  }
+
+  /**
+   * Sets the UserStore used by this servlet. This function provides a common setup method for use
+   * by the test framework or the servlet's init() function.
+   */
+  void setUserStore(UserStore userStore) {
+    this.userStore = userStore;
+  }
+
+  // Forwards the request to activity-feed.jsp.
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException, ServletException {
+    request.getRequestDispatcher("/WEB-INF/view/activityfeed.jsp").forward(request, response);
```

Comment:
Can we get the user here?

Reply:
So we can verify that we can use the userStore properly

Reply:
UserStore#getUser actually. So getting the user from the userstore itself. Similar to how it is done in ChatServlet#doPost, and ConversationServlet#doPost.

Reply:
Resolved.